src/components/NonProfitApply/NonProfitApply.js

```diff
@@ -108,11 +143,19 @@ export default function NonProfitApply() {
 
 
   var loginCallToAction = <LoginOrRegister 
-      introText={"Whoa there - in order to submit this nonprofit form, you need to login or create an account first."} 
+      introText={"Please consider logging in or creating an account. You'll need a Slack account for the hackathon."} 
```

Comment:
Now we're showing this after someone submits a form to encourage them to create a Slack account instead of creating that friction before they submit the form